scripts/download-env.sh

```diff
@@ -31,6 +31,20 @@ if [ ! -z "$SETTINGS_FILE" -o ! -z "$XILINX" ]; then
 	exit 1
 fi
 
+# Conda does not support ' ' in the path (it bails early).
+if echo "${SETUP_DIR}" | grep -q ' '; then
+  echo "You appear to have whitespace characters in the path to this script."
+  echo "Please move this repository to another path that does not contain whitespace."
+  exit 1
+fi
+
+# Conda does not support ':' in the path (it fails to install python).
+if echo "${SETUP_DIR}" | grep -q ':'; then
+  echo "You appear to have ':' characters in the path to this script."
+  echo "Please move this repository to another path that does not contain this character."
+  exit 1
+fi
```

Comment:
Could you add these checks to `enter-env.sh` as well? You should be able to do; `vimdiff download-env.sh enter-env.sh` and see how they are mostly trying to be the same.